tests/foreman/cli/test_contentviews.py

```diff
@@ -142,6 +142,46 @@ def setUp(self):
                 {u'organization-id': TestContentView.org['id']},
                 cached=True)
 
+    def test_bz1184442(self):
```

Comment:
[BZ1184442](https://bugzilla.redhat.com/show_bug.cgi?id=1184442) is already closed.